src/source/view/transitions/BaseTransition.bs

```diff
@@ -90,6 +94,10 @@ namespace mv.transitions
 
         if m.owner.isShown
           m.target@._setShown(true)
+          if m.owner.pushedView <> invalid
```

Comment:
why guard?

Reply:
true .  i was thinking that other "owners" might not have the fields, but it doesn't matter anyhow. making the changes now